specification/deviceregistry/DeviceRegistry.Management/client.tsp

```diff
@@ -3,4 +3,9 @@ import "@azure-tools/typespec-client-generator-core";
 
 using Azure.ClientGenerator.Core;
 
+@@clientNamespace(Microsoft.DeviceRegistry,
+  "com.azure.resourcemanager.deviceregistry",
+  "java"
+);
 @@clientName(Microsoft.DeviceRegistry, "DeviceRegistryMgmt", "python");
+@@clientName(Microsoft.DeviceRegistry, "DeviceRegistryClient", "java");
```

Comment:
ManagementClient

Reply:
Fixed in the [commit#58b279b](https://github.com/Azure/azure-rest-api-specs/pull/31805/commits/58b279b0a5e2387b78a712b5de53caf8053cb015)